src/editor/extended_python_generator.ts

```diff
@@ -248,7 +255,7 @@ export class ExtendedPythonGenerator extends PythonGenerator {
       exportedBlocks.push(callFunctionBlock);
     }
 
-    const allVariables = workspace.getAllVariables();
+    const allVariables = workspace.getVariableMap().getAllVariables();
```

Comment:
Isn't this the same thing you had to revert earlier?

Reply:
getAllVariables is ok to call after getVariableMap(). getVariableUsesById is not ok to call after getVariableMap().